scripts/coverage

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
pytest should already be configured to generate code coverage via the `pyproject.toml` configuration. Was that not working for your environment?

Reply:
if pytest is configured to generate code coverage debugging in vscode does not work -- I think this is a relevant issue https://code.visualstudio.com/docs/python/testing

> Note If you have the pytest-cov coverage module installed, VS Code doesn't stop at breakpoints while debugging because pytest-cov is using the same technique to access the source code being run. To prevent this behavior, include --no-cov in pytestArgs when debugging tests, for example by adding "env": {"PYTEST_ADDOPTS": "--no-cov"} to your debug configuration. (See [Debug Tests](https://code.visualstudio.com/docs/python/testing#_debug-tests) above about how to set up that launch configuration.) (For more information, see [Debuggers and PyCharm](https://pytest-cov.readthedocs.io/en/latest/debuggers.html) in the pytest-cov documentation.)